### DISCUSSION TITLE
[enhancement] enter after complete task gives bullet not new open task

### DISCUSSION THREAD

**Discussion by eleanorkonik** [vslinko/obsidian-outliner#250] [Ideas]
I don't know if this is within scope for this plugin, but I figured I'd ask here since it's the closest plugin I was able to find. Would it be possible to add an option to "continue with current type" ? Most outliners have basically 3 kinds of "bullets" — `*` or `-`, `- [ ]` and `- [x]` and generally people going from a task (the latter two) want a new _open_ task after it. 

Obsidian supports bullet journal style task markers, i.e. `- [>]` and generally you wouldn't necessarily want to have a bunch of those in a row, you'd want an open task thing there, too. Which is why Licat won't add the thing I (and others, @slrvb wants this too) find myself _itching_ for some way to do multiple:

- [I] ideas
- [I] ideas

or 

- [n] notes
- [n] notes in a row. 

In github's little outliner, if you hit enter after `- [ ]` the outline gives you `-` which is nice because then I can just type ` [n]` and not disrupt my flow by having to backspace. 

A _lot_ of my outlines are peppered with custom checkboxes, which are supported by [themes like Sanctum and ITS Theme](https://github.com/jdanielmourao/obsidian-sanctum/blob/eb25c4de5d82440f8be7e52dc6c23e918e39abce/snippets/Sanctum%20Custom%20Checkboxes.css). I also find myself marking "accomplishments" and doing "pro con lists" where I want to do a lot of stuff like

- [p]
- [c]
- [p]

and Obsidian's current outliner behavior makes this tricky becauase I have to backspace and it's obnoxious. 

Would you be able to make a setting so that if I'm hitting enter after a `- [t] task` it just gives me a `-` or `*` instead of `- [ ]`?

**Comment by vslinko** [comment]
@eleanorkonik Thank you for your feedback! Yes, I can add the setting.
